packages/client/src/runtime/index.ts

```diff
@@ -17,6 +17,7 @@ export type { DecimalJsLike } from './utils/decimalJsLike'
 export { findSync } from './utils/find'
 export { NotFoundError } from './utils/rejectOnNotFound'
 export { warnEnvConflicts } from './warnEnvConflicts'
+export { default as debug } from '@prisma/debug'
```

Comment:
I think this also works
```suggestion
export { Debug } from '@prisma/debug'
```

Reply:
Done!